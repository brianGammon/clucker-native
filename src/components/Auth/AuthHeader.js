import React from 'react';
import { View, Text, H1 } from 'native-base';
import { Image } from 'react-native';
import styles from './styles';

const AuthHeader = () => (
  <View style={styles.headerContainer}>
    <Image
      style={styles.headerImage}
      source={require('../../assets/chicken-sm.png')}
    />
    <View style={styles.headerCopy}>
      <H1 style={styles.headerTitle}>Clucker</H1>
      <Text style={styles.headerSubTitle}>
        Your Backyard Chicken Egg Tracker
      </Text>
    </View>
  </View>
);

export default AuthHeader;
