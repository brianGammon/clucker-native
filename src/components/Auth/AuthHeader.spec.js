import React from 'react';
import AuthHeader from './AuthHeader';

describe('AuthHeader', () => {
  test('renders', () => {
    const wrapper = shallow(<AuthHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
