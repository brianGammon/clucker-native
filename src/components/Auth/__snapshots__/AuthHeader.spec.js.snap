// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthHeader renders 1`] = `
<Styled(ViewNB)
  style={
    Object {
      "flexDirection": "row",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/chicken-sm.png",
      }
    }
    style={
      Object {
        "height": 80,
        "width": 80,
      }
    }
  />
  <Styled(ViewNB)
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <Styled(H1)
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      Clucker
    </Styled(H1)>
    <Styled(Text)
      style={
        Object {
          "color": "grey",
        }
      }
    >
      Your Backyard Chicken Egg Tracker
    </Styled(Text)>
  </Styled(ViewNB)>
</Styled(ViewNB)>
`;
